pegasus/cache/i18n/ru-RU.yml

```diff
@@ -483,9 +483,10 @@
   homepage_header_donors: Code.org благодарит своих помощников
   homepage_donors_seall: Просмотреть всё
   homepage_donors_donate: Пожертвовать
-  homepage_below_hero_announcement_text: We care about your privacy and are making
-    some updates to our privacy policy, effective May 25.
-  homepage_below_hero_announcement_link_text: Узнать больше
+  homepage_below_hero_announcement_text: Мы соблюдаем вашу конфиденциальность и
+    вносим некоторые обновления в нашу политику конфиденциальности, которые вступают
+    в силу с 25 мая.
+  homepage_below_hero_announcement_link_text: Подробнее
```

Comment:
Second instance where the string was already translated. Professional translations is briefer (better for buttons) and according to Google Translate says "Learn More" while the volunteer translation is more like "to learn more." Will take the professional translation.